frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-broker-actions.feature

```diff
@@ -0,0 +1,123 @@
+@knative-eventing
+Feature: Perform actions on Broker
+              As a user, I should be able to perform actions on broker
+        
+        
+        Background:
+            Given user has installed OpenShift Serverless Operator
+              And user has created Knative Eventing CR
+              And user has created or selected namespace "aut-eventing-broker-actions"
+              And user is at Topology page
+        
+        
+        @regression @to-do @odc-5030
+        Scenario: Context Menu for Broker: KC-01-TC01
+            Given user has created broker "default" in application groupings "app"
```

Comment:
Keep this in the Background statement as you are using it in the entire file

Reply:
But it will create for every scenario @sanketpathak and that will cause the error. Also, I am using the same broker for other scenarios.